src/client/automation/playback/press/shortcuts.js

```diff
@@ -503,46 +514,31 @@ function enter (element) {
     return Promise.resolve();
 }
 
-function focusNextElement (element) {
-    return new Promise(resolve => {
-        if (domUtils.isSelectElement(element)) {
-            selectElement.collapseOptionList();
-            resolve();
-        }
-
-        const nextElement = domUtils.getNextFocusableElement(element);
-
-        if (!nextElement)
-            resolve();
-
-        focusBlurSandbox.focus(nextElement, () => {
-            if (domUtils.isTextEditableInput(nextElement))
-                textSelection.select(nextElement);
+function isRadioButtonNavigationRequired (element) {
+    return domUtils.isRadioButtonElement(element) && !browserUtils.isFirefox;
+}
 
-            resolve();
+function focusAndCheckNextRadioButton (element, reverse) {
+    return focusNextElement(element, reverse, false)
+        .then(focusedElement => {
+            if (focusedElement)
+                focusedElement.checked = true;
         });
-    });
 }
 
-function focusPrevElement (element) {
-    return new Promise(resolve => {
-        if (domUtils.isSelectElement(element)) {
-            selectElement.collapseOptionList();
-            resolve();
-        }
-
-        const prevElement = domUtils.getNextFocusableElement(element, true);
+function focusNextElement (element, reverse, skipRadioGroups) {
+    if (!element)
+        return Promise.resolve();
 
-        if (!prevElement)
-            resolve();
-
-        focusBlurSandbox.focus(prevElement, () => {
-            if (domUtils.isTextEditableInput(prevElement))
-                textSelection.select(prevElement);
+    if (domUtils.isSelectElement(element))
```

Comment:
Should we focus next element if `domUtils.isSelectElement(element) === true` ?

Reply:
Yes, we have test for this